index.js

```diff
@@ -29,14 +27,14 @@ function main(fn, fp, data, opts) {
 		});
 	}
 
-	var json = JSON.stringify(data, opts.replacer, opts.indent) + '\n';
+	const json = JSON.stringify(data, opts.replacer, opts.indent) + '\n';
```

Comment:
I believe when adding `esnext: true` to the XO configuration, it will force you to change it. You could make it a separate constant. Not sure what @sindresorhus prefers.